File: 2 курс/Веб-программирование/lab4/front/src/app/dto/AuthResponse.ts
export class AuthResponse {
  constructor(
    private _isValid: boolean,
    private _message: string,
    private _id: string
  ) {}

  get isValid(): boolean {
    return this._isValid;
  }

  get message(): string {
    return this._message;
  }

  get id(): string {
    return this._id;
  }

  set isValid(value: boolean) {
    this._isValid = value;
  }

  set message(value: string) {
    this._message = value;
  }

  set id(value: string) {
    this._id = value;
  }
}

File: 2 курс/Веб-программирование/lab4/backend/src/main/java/dto/AuthResponse.java
package dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class AuthResponse {
    private String message;
    private String id;
    private boolean isValid;

    public AuthResponse(String message, String id) {
        this.isValid = true;
        this.message = message;
        this.id = id;
    }

    public AuthResponse(boolean isValid, String message) {
        this.isValid = isValid;
        this.message = message;
    }

}


